src/ntfs/mod.rs

```diff
@@ -242,23 +242,21 @@ mod tests {
     #[test]
     fn feed_infos_serialization_deserialization() {
         let mut feed_infos = BTreeMap::default();
-        feed_infos.insert("feed_license".to_string(), "".to_string());
-        feed_infos.insert("ntfs_version".to_string(), "0.3".to_string());
-        feed_infos.insert("feed_creation_date".to_string(), "20181004".to_string());
+        feed_infos.insert("fusio_version".to_string(), "1.10.89.209".to_string());
+        feed_infos.insert(
+            "fusio_url".to_string(),
+            "http://vip-fusio-ihm.FR-IDF-OPEN.prod.canaltp.fr/".to_string(),
+        );
         let mut collections = Collections::default();
-
         test_in_tmp_dir(|path| {
-            write::write_feed_infos(path, &feed_infos).unwrap();
+            write::write_feed_infos(path, &feed_infos, &collections.datasets).unwrap();
             read::manage_feed_infos(&mut collections, path).unwrap();
-            let info_params: Vec<_> = collections.feed_infos.keys().collect();
-            // test that feed infos are ordered by info_param
-            assert_eq!(
-                info_params,
-                ["feed_creation_date", "feed_license", "ntfs_version"]
-            );
+            let c_feed_infos: BTreeSet<_> = collections.feed_infos.iter().collect();
```

Comment:
I would use a HashSet but it's a detail

Reply:
Better with BTreeSet because it keeps order (already try with HashSet)